.github/workflows/equinix_k8s_flow.yml

```diff
@@ -89,12 +89,14 @@ jobs:
       - name: Install Prometheus
         run: |
           git clone --depth 1 https://github.com/prometheus-operator/kube-prometheus; cd kube-prometheus;
-          kubectl apply --server-side -f manifests/setup
+          kubectl apply --server-side --validate=false -f manifests/setup
           until kubectl get servicemonitors --all-namespaces ; do date; sleep 1; echo ""; done
-          kubectl apply -f manifests/
+          kubectl apply --validate=false -f manifests/
           kubectl wait deployments -n monitoring prometheus-adapter --for=condition=available --timeout 3m
-          kubectl port-forward -n monitoring svc/prometheus-k8s 9090:9090 &
 
+      - name: Label workers
+        run: |
+          kubectl label node $(oc get nodes --selector=node-role.kubernetes.io/control-plane!= --no-headers -o custom-columns=":metadata.name") node-role.kubernetes.io/worker=
```

Comment:
oc is not installed yet, can you use kubectl?

Reply:
yeah, typo on my part while debugging.